### DISCUSSION TITLE
I don't know

### DISCUSSION THREAD

**Discussion by OHoRaaa** [lucidrains/DALLE2-pytorch#129] [Q&A]
Output image is just noise image, why? 😭

**Comment by alrik11es** [comment]
I have the same problem. Looks like something is missing.

**Comment by twisted-nematic57** [comment]
How did you even get it to run? I got an error:
```
Traceback (most recent call last):
  File "D:/dalle.py", line 44, in <module>
    diffusion_prior = DiffusionPrior(
TypeError: DiffusionPrior.__init__() got an unexpected keyword argument 'sample_timesteps'
```

**Comment by u1ug** [comment]
> Output image is just noise image, why? 😭

Did you trained it on real examples? In example code we just feed model with random noise
